### DISCUSSION TITLE
Unable to upgrade to the latest version of Tailwind CSS, PostCSS and Autoprefixer

### DISCUSSION THREAD

**Discussion by toxifiedmatrix** [tailwindlabs/tailwindcss#4063] [Help]
I am unable to upgrade from the Tailwind CSS v2.0.3 to v2.1.1 along with PostCSS and Autoprefixer. Currently I am on Laravel 8, Laravel Breeze, Inertia JS, Vue JS, I am trying to upgrade to the latest version by running `npm update` and `npm update tailwindcss` but I am unable to upgrade to the latest version. Please guide me how can I achieve the said above.

**Comment by MartinHughes-BPC** [comment]
Try: 

    npm install -D tailwindcss@latest postcss@latest autoprefixer@latest

That should do the trick. I think.

**Reply by simonswiss** [reply]
Yep, I was also going to recommend this ☝️
